src/__tests__/createActions-test.js

```diff
@@ -0,0 +1,29 @@
+import { createActions } from '../';
+
+describe('createActions()', () => {
+  describe('resulting action creator map', () => {
+    const types = {
+      ACTION: 'ACTION',
+      MY_LONG_ACTION: 'MY_LONG_ACTION'
+    };
+
+    it(`apply a default string transformation function to action types
+        when mapping to action creators`, () => {
+      const actionCreators = createActions(types);
+
+      expect(actionCreators).to.have.property('action');
+      expect(actionCreators).to.have.property('myLongAction');
+    });
```

Comment:
What should the output be if you camelize an already camelized string?

As it is, if you camelize FOO_BAR, and camelize the output of that, you get something different.